hibernate-core/src/main/java/org/hibernate/id/uuid/UuidGenerator.java

```diff
@@ -81,6 +83,12 @@ private static UuidValueGenerator determineValueGenerator(
 			else if ( config.style() == TIME ) {
 				return new CustomVersionOneStrategy();
 			}
+			else if ( config.style() == VERSION_6 ) {
```

Comment:
Looking at this closer now though, I wonder if we should reuse the same instance of these (singleton) rather than creating one for each id?

Reply:
Perhaps, yes. This will prevent two entities to have equal id, but this is something that is good to avoid, anyway.

Reply:
Looks great, thanks!